src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java

```diff
@@ -0,0 +1,196 @@
+        GridBagConstraints gbc_lblFormatColor = new GridBagConstraints();
+        gbc_lblFormatColor.anchor = GridBagConstraints.EAST;
+        gbc_lblFormatColor.insets = new Insets(0, 0, 5, 5);
+        gbc_lblFormatColor.gridx = 1;
+        gbc_lblFormatColor.gridy = 5;
+        add(lblFormatColor, gbc_lblFormatColor);
+
+        comboBoxColorFormat = new JComboBox();
+        comboBoxColorFormat.setModel(new DefaultComboBoxModel(new String[]{"RGB", "RGBA", "GRAY", "BILEVEL"}));
+        GridBagConstraints gbc_comboBoxColorFormat = new GridBagConstraints();
+        gbc_comboBoxColorFormat.insets = new Insets(0, 0, 5, 0);
+        gbc_comboBoxColorFormat.fill = GridBagConstraints.HORIZONTAL;
+        gbc_comboBoxColorFormat.gridx = 2;
+        gbc_comboBoxColorFormat.gridy = 5;
+        add(comboBoxColorFormat, gbc_comboBoxColorFormat);
+
+        JLabel lblDpi = new JLabel("DPI:");
+        GridBagConstraints gbc_lblDpi = new GridBagConstraints();
+        gbc_lblDpi.anchor = GridBagConstraints.EAST;
+        gbc_lblDpi.insets = new Insets(0, 0, 5, 5);
+        gbc_lblDpi.gridx = 1;
+        gbc_lblDpi.gridy = 6;
+        add(lblDpi, gbc_lblDpi);
+
+        comboBoxDpi = new JComboBox();
+        comboBoxDpi.setModel(new DefaultComboBoxModel(new String[]{"72", "96", "160", "300", "600", "4000"}));
+        GridBagConstraints gbc_comboBoxDpi = new GridBagConstraints();
+        gbc_comboBoxDpi.insets = new Insets(0, 0, 5, 0);
+        gbc_comboBoxDpi.fill = GridBagConstraints.HORIZONTAL;
+        gbc_comboBoxDpi.gridx = 2;
+        gbc_comboBoxDpi.gridy = 6;
+        add(comboBoxDpi, gbc_comboBoxDpi);
+
+        JLabel lblThumbnail = new JLabel("Thumbnail:");
+        GridBagConstraints gbc_lblThumbnail = new GridBagConstraints();
+        gbc_lblThumbnail.anchor = GridBagConstraints.EAST;
+        gbc_lblThumbnail.insets = new Insets(0, 0, 5, 5);
+        gbc_lblThumbnail.gridx = 1;
+        gbc_lblThumbnail.gridy = 7;
+        add(lblThumbnail, gbc_lblThumbnail);
+
+        checkBoxThumbnail = new JCheckBox("");
+        GridBagConstraints gbc_checkBoxThumbnail = new GridBagConstraints();
+        gbc_checkBoxThumbnail.anchor = GridBagConstraints.WEST;
+        gbc_checkBoxThumbnail.insets = new Insets(0, 0, 5, 5);
+        gbc_checkBoxThumbnail.gridx = 2;
+        gbc_checkBoxThumbnail.gridy = 7;
+        add(checkBoxThumbnail, gbc_checkBoxThumbnail);
+
+        JLabel lblThumbnailFormat = new JLabel("Format: ");
+        GridBagConstraints gbc_lblThumbnailFormat = new GridBagConstraints();
+        gbc_lblThumbnailFormat.anchor = GridBagConstraints.EAST;
+        gbc_lblThumbnailFormat.insets = new Insets(0, 0, 5, 5);
+        gbc_lblThumbnailFormat.gridx = 2;
+        gbc_lblThumbnailFormat.gridy = 7;
+        add(lblThumbnailFormat, gbc_lblThumbnailFormat);
+
+        comboBoxThumbnailFormat = new JComboBox();
+        comboBoxThumbnailFormat.setModel(new DefaultComboBoxModel(new String[]{"bmp", "png", "jpg"}));
```

Comment:
All previous implemented comboBox show options in uppercase.

Reply:
Done